hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -551,7 +551,88 @@
       default:   "5",
     },
   ],
+  features: [
+    {
+     name: "FLASH_CTRL.ESCALATION"
+     desc:'''Flash controller has two sources of escalation.
+     1. Global escalation : This is initiated by the LC (Lifecycle Controller) through the use of lc_escalate_en.
+     2. Local escalation : This is activated in response to standard faults detected within the flash.
+                           These faults are monitored through flash_ctrl.STD_FAULT_STATUS.
+     '''
+    }
+    {
+      name: "FLASH_CTRL.FETCH_CODE"
+      desc: '''If SW program flash_ctrl.EXEC to 0xA26A38F7, code fetch from flash device is allowed.
+            All test_rom test uses this features. No dedicated test is required.
+      '''
+    }
+    {
+      name: "FLASH_CTRL.INFO.CREATOR_PARTITION"
+      desc: '''This partition stores creator root seed.
+            It is accessed by the flash controller after scramble key is received.
+            SW can read or program the contents with both conditions are satisfied.
+              1. Life cycle state should be either one of DEV / PROD / PROD_END / RMA.
+              2. The device has not yet been personalized.
+            '''
+    }
+    {
+      name: "FLASH_CTRL.INFO.ISOLATED_PARTITION"
+      desc: '''This partition can hold extra manufacturing details.
+            SW can read or program the contents when life cycle state is either of of PROD / PROD_END / RMA.
+            SW can only program the contents when life cycle state is TEST_UNLOCKED* or DEV.
```

Comment:
I think this should be:
1. only PROGRAMMABLE in TEST_UNLOCKED*, and
2. only READABLE in DEV / PROD / PROD_END / RMA.

See https://opentitan.org/book/hw/ip/flash_ctrl/#isolated-information-partitions

Note, I have noticed this is not how things currently are. Namely, from experimentation, it seems the page is READABLE and PROGRAMMABLE in TEST_UNLOCKED*, not READABLE in DEV, and only READABLE in PROD / PROD_END / RMA. Was meaning to file an issue about this ...

Reply:
@timothytrippel 
This was captured from [lc_ctrl_spec](https://opentitan.org/book/hw/ip/lc_ctrl/doc/theory_of_operation.html#life-cycle-access-control-signals).
Can you clarify which one needs to be updated?

Reply:
Filed issue #19679